src/components/Wrappers/withClaimReturned.js

```diff
@@ -15,6 +66,180 @@ const withClaimReturned = WrappedComponent => class withClaimReturnedComponent e
     };
   }
 
+  refundTransfers = async () => {
+    const getAccounts = (itemBarcode, userId) => {
+      const {
+        mutator: {
+          accounts: {
+            GET,
+          },
+        }
+      } = this.props;
+      const lostStatus = 'Lost item fee';
+      const processingStatus = 'Lost item processing fee';
+      const pathParts = [
+        'accounts?query=',
+        `barcode=="${itemBarcode}"`,
+        ` and userId=="${userId}"`,
+        ` and (feeFineType=="${lostStatus}"`,
+        ` or feeFineType=="${processingStatus}")`
+      ];
+      const path = pathParts.reduce((acc, val) => acc + val, '');
+      return GET({ path });
+    };
+
+    const setPaymentStatus = record => {
+      const updatedRec = cloneDeep(record);
+      updatedRec.paymentStatus.name = 'Suspended claim returned';
+      return updatedRec;
+    };
+
+    const persistAccountRecord = record => {
+      const {
+        mutator: {
+          activeAccount: {
+            update,
+          },
+          accounts: {
+            PUT,
+          }
+        }
+      } = this.props;
+      update({ id: record.id });
+      return PUT(record);
+    };
+
+    const getAccountActions = account => {
+      const {
+        mutator: {
+          feefineactions: {
+            GET,
+          }
+        }
+      } = this.props;
+      const path = `feefineactions?query=(accountId==${account.id})&orderBy=dateAction&order=desc`;
+      return GET({ path });
+    };
+
+    const filterTransferredActions = actions => actions
+      .filter(
+        record => record.typeAction && record.typeAction.startsWith('Transferred')
+      );
+
+    const getLastBalance = actions => (
+      actions.length > 0
+        ? orderBy(actions, ['dateAction'], ['desc'])[0].balance
+        : 0.0
+    );
+
+    const createRefundActionTemplate = (account, transferredActions, type) => {
+      const {
+        okapi: {
+          currentUser: {
+            id: currentUserId,
+            curServicePoint: {
+              id: servicePointId
+            }
+          },
+        },
+      } = this.props;
+      const orderedActions = orderBy(transferredActions, ['dateAction'], ['desc']);
+      const now = moment().format();
+      const sign = type.startsWith('Credited') ? -1 : 1;
+      const amount = sign * transferredActions.reduce((acc, record) => acc + record.amountAction, 0.0);
+      const transactionVerb = type.startsWith('Credited')
+        ? 'Refund'
+        : 'Refunded';
+      return {
+        dateAction: now,
+        typeAction: type,
+        comments: '',
+        notify: false,
+        amountAction: amount,
+        balance: 0,
+        transactionInformation: `${transactionVerb} to ${orderedActions[0].paymentMethod}`,
```

Comment:
I'm concerned `transactionInformation` won't localize correctly because of the way it is structured. Could/should the verb and method be handled separately?

Reply:
This is how the transaction information is handled:
![imagen](https://user-images.githubusercontent.com/39131594/103563120-d4086600-4e81-11eb-9412-ca1cc6eb99e8.png) 
but we can ask Holly Mistlebauer